lang/python/python-requests-oauthlib/Makefile

```diff
@@ -10,31 +8,68 @@ include $(TOPDIR)/rules.mk
 PKG_NAME:=python-requests-oauthlib
 PKG_VERSION:=1.2.0
 PKG_RELEASE:=1
+
+PKG_MAINTAINER:=Eneas U de Queiroz <cote2004-github@yahoo.com>
 PKG_LICENSE:=ISC
+PKG_LICENSE_FILES:=LICENSE
 
 PKG_SOURCE:=requests-oauthlib-$(PKG_VERSION).tar.gz
 PKG_SOURCE_URL:=https://files.pythonhosted.org/packages/source/r/requests-oauthlib
 PKG_HASH:=bd6533330e8748e94bf0b214775fed487d309b8b8fe823dc45641ebcd9a32f57
-PKG_BUILD_DIR:=$(BUILD_DIR)/requests-oauthlib-$(PKG_VERSION)
+
+PKG_BUILD_DIR:=$(BUILD_DIR)/$(BUILD_VARIANT)-requests-oauthlib-$(PKG_VERSION)
 
 include $(INCLUDE_DIR)/package.mk
 include ../python-package.mk
+include ../python3-package.mk
 
-define Package/python-requests-oauthlib
+PKG_UNPACK:=$(HOST_TAR) -C $(PKG_BUILD_DIR) --strip-components=1 -xzf $(DL_DIR)/$(PKG_SOURCE)
+
+define Package/python-requests-oauthlib/Default
   SUBMENU:=Python
   SECTION:=lang
   CATEGORY:=Languages
-  MAINTAINER:=Eneas U de Queiroz <cote2004-github@yahoo.com>
-  TITLE:=OAuthlib authentication support for Requests.
+  TITLE:=OAuthlib auth for Requests
   URL:=https://github.com/requests/requests-oauthlib
-  DEPENDS:=+python +python-requests +python-oauthlib +python-cryptography +python-pyjwt
+endef
+
+define Package/python-requests-oauthlib
+  $(call Package/python-requests-oauthlib/Default)
+  DEPENDS:= \
+	+PACKAGE_python-requests-oauthlib:python \
+	+PACKAGE_python-requests-oauthlib:python-cryptography \
+	+PACKAGE_python-requests-oauthlib:python-oauthlib \
+	+PACKAGE_python-requests-oauthlib:python-pyjwt \
+	+PACKAGE_python-requests-oauthlib:python-requests
```

Comment:
(This is somewhat unrelated to this specific PR, apologies in advance.)

It looks like from [setup.py](https://github.com/requests/requests-oauthlib/blob/v1.2.0/setup.py#L47) that only oauthlib and requests are dependencies... I did a quick search and it looks like cryptography and pyjwt are only used in [one test](https://github.com/requests/requests-oauthlib/blob/v1.2.0/tests/test_oauth1_session.py)?

If this is the case, I would prefer to leave cryptography and pyjwt out as required dependencies (can perhaps add a note for those who want to run the test to install those packages).

Reply:
It is 100% relevant, and I will change right away.  You're right, it is only used for that test, and we don't ship them anyway, so I won't leave a note.  I didn't have much experience when I first added the package, and it probably barked at me when I tried to run the tests.  At least now I know a little more, so let me make this right.  Thanks for pointing this out!